### PR TITLE
Radio host room

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Makes a radio host room, because why not.

## Why It's Good For The Game

Has a security camera console for reporting on things around the station as well as 4 command radios which can access higher frequencies.

## Changelog
:cl:
add: Radio host room
/:cl:

### CODE REVIEW

**Comment by 4rchibald** [comment]
Would it be possible to give radio host intercom or headset that uses different colour in chat? It would be easier to distinguish and in result easier follow or ignore

File: code/modules/clothing/outfits/plasmaman.dm [hunk 1]
```diff
@@ -16,6 +16,12 @@
 	head = /obj/item/clothing/head/foilhat/plasmaman
 	uniform = /obj/item/clothing/under/plasmaman
 
+/datum/outfit/plasmaman/radio
+	name = "Radiohost Plasmaman"
+
+	head = /obj/item/clothing/head/helmet/space/plasmaman/hat
```
Comment (zeskorion) [inline code/modules/clothing/outfits/plasmaman.dm:22]: Iirc you're using the magician's set here. I'd at least change the hat, to a normal service envirohelm

File: code/modules/mapping/random_rooms.dm [hunk 2]
```diff
@@ -907,4 +907,12 @@
 	mappath = "_maps/RandomRooms/10x10/sk_rdm106_sanitarium.dmm"
 	centerspawner = FALSE
 	template_height = 10
-	template_width = 10
\ No newline at end of file
+	template_width = 10
+
+/datum/map_template/random_room/sk_rdm106
```
Comment (zeskorion) [inline code/modules/mapping/random_rooms.dm:912]: 10x10s all have deflated weighting. This room will pop up way too often for this niche gimmick- put it around 4-6

**Review by zeskorion — COMMENTED** [review]
Also- most gimmick jobs have two rooms (as of last maint pr )- one large, one small. Please map a small hosting booth

**Comment by Bastian0930** [comment]
This is like the space ruin. You should add anything given to this gimmick to the pirate radio ruin, too!

**Pull request closed without merging**